addon/package.json

```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "array-includes": "^3.0.1",
     "exclude": "^1.0.0",
-    "xtend": "^4.0.0"
+    "idea-town": "0.0.1"
```

Comment:
Maybe it's not possible, but could this npm package be part of the main idea-town repo in a subdirectory rather than a new github repo? We just spent a bit of gymnastics to stand up a cluster of repos and then merged them. It would be great if we could keep everything still all together.